pegasus/routes/v2_section_routes.rb

```diff
@@ -1,4 +1,5 @@
 # Get the set of sections owned by the current user
+# DEPRECATED: Use GET /dashboardapi/sections instead
```

Comment:
Yes, we'd like to deprecate (and eventually remove) this whole file and reimplement it within dashboard.  It follows the Rails [resource routing](http://guides.rubyonrails.org/routing.html) idiom and will be better implemented there in terms of our models.  Before we actually remove this file though, we plan to instrument these routes with New Relic events and monitor for a little while to make sure nobody (including nobody external) is still using them.